### DISCUSSION TITLE
Control the width of an SVG

### DISCUSSION THREAD

**Discussion by arkadianriver** [dita-ot/dita-ot#4515] [Q&A]
In a PDF, is there a way to include an SVG and have it be the width of the page or column, without having to add a hard-coded width to an `<image>` tag or in the case of an `<svgref>` to the SVG file itself? I believe `expanse="column"` on the parent `<fig>` only determines from where to start the rendering of the image (column margin, page margin, etc.) and not its width.

**Comment by stefan-jung** [comment]
You can set the `width` and `height` on the **`<svg>`** root element. You can do this with Inkscape in the document settings.

**Reply by arkadianriver** [reply]
Is there another way besides editing the SVG file? Say, if there are tons of files and they are used in more than one sized output.
